src/site/pages/upgrade-guide.md

```diff
@@ -0,0 +1,223 @@
+# PatternFly 4 Upgrade Guide
+
+Upgrading your environment from PatternFly 3 to PatternFly 4.
+
+**Important**
+
+When upgrading from a PatternFly 3 environment to a PatternFly 4 environment, you will need to do the following:
+
+- Verify that your application does not need IE 11 support. If it does, you cannot upgrade to PatternFly 4.
+- Check your build process. The steps for basic HTML, Gulp and Webpack differ and have different requirements.
+- Determine if you need to run PatternFly 3 and PatternFly 4 together. If you do, you will need to update the following configuration in the `src/patternfly/utilities/variables.scss` file to state `true`.
+
+```scss
+$pf-global--load-pf-3: true !default;
```

Comment:
it's not this one that people need to change, it's:

`$pf-global--enable-reset: false !default;`

`$pf-global--load-pf-3` loads pf3 for development to make sure there are no styles leaks.

Since pf3 already has a reset, loading pf4 reset will generate conflicts with pf3.
PF4 can work perfectly fine with pf3 reset, therefore you'd want to turn it off.